packages/ibm-products-styles/src/components/SidePanel/_side-panel.scss

```diff
@@ -77,12 +77,14 @@ $action-set-block-class: #{c4p-settings.$pkg-prefix}--action-set;
       @include setPanelSize($size_value);
       .#{$block-class}__title-container.#{$block-class}__title-container--no-title-animation,
       .#{$block-class}__subtitle-text.#{$block-class}__subtitle-text-no-animation,
-      .#{$block-class}__action-toolbar.#{$block-class}__action-toolbar-no-animation,
-      .#{$block-class}__actions-container.#{$action-set-block-class}--#{$size} {
+      .#{$block-class}__action-toolbar.#{$block-class}__action-toolbar-no-animation {
         @include setPanelSize($size_value);
       }
     }
   }
+  .#{$block-class}__actions-container {
+    width: 100%;
+  }
```

Comment:
Before: the actions container is the "same size" as the side panel. 
- this worked when using absolute values like`rems` ("Make the side panel and actions container 20rems wide"), 
- but broke when using relative values like `%` ("Make the side panel 75% the width of the page, and the actions container 75% of the side panel"). This issue only happens where `size` is `2xl`.

After: the actions container is a static 100% the width of the side panel.